src/component/Restaurants.js

```diff
@@ -1,35 +1,57 @@
 import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { RestaurantListItem } from "./RestaurantListItem";
 
-const api = "/cities";
+import thumbnail1Src from "place-my-order-assets/images/1-thumbnail.jpg";
+import thumbnail2Src from "place-my-order-assets/images/2-thumbnail.jpg";
+import thumbnail3Src from "place-my-order-assets/images/3-thumbnail.jpg";
+import thumbnail4Src from "place-my-order-assets/images/4-thumbnail.jpg";
 
 /**
  * Displays information about available restaurants.
  * @returns {JSX.Element}
  */
 function Restaurants (props) {
-    const [region, setRegion] = useState("");
     const [cities, setCities] = useState({});
-    const [city, setCity] = useState("");
+    const [city, setCity] = useState(DEFAULT_SELECT_VALUE);
+    const [region, setRegion] = useState(DEFAULT_SELECT_VALUE);
+    /** @type {[RestaurantRegionData, (value: RestaurantRegionData) => void]} */
```

Comment:
What is this line? `/** @type {[RestaurantRegionData, (value: RestaurantRegionData) => void]} */`

Is this something we should be doing everywhere?

Reply:
Just a jsdoc comment. It helps vscode with code completion. I'd prefer to use TypeScript which would also provide the information but since the point of the course is to introduce React I think typescript would just be another prerequisite barrier and didn't include it.